src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -710,6 +709,104 @@ transactionFormatResultImpl(Transaction::pointer tpTrans, unsigned apiVersion)
 
 //------------------------------------------------------------------------------
 
+XRPAmount
+getBaseFee(Application const& app, Config const& config, Json::Value tx)
```

Comment:
This function is not exported, please make it static or put into anonymous namespace.